Financial-Times/n-myft-ui
# UG-936 Display notification after unsaving article

### Description
After a user unsaves an article, a notification is displayed. See [designs](https://www.figma.com/file/09mlE9M0XVG5fIZy99tls2/(Final)-Professor-Engagement?node-id=758%3A7466) (Frame 245 & Frame 249).

### Ticket
https://financialtimes.atlassian.net/browse/UG-936

### How To Test
1) clone `next-article` and run `npm install build` 
2) Make sure you’re logged in on Vault and run `next-article` with `npm start`
3) Switch on the manageArticleLists flag on [Toggler](https://toggler.ft.com/#manageArticleLists)
4) visit https://local.ft.com:5050/
5) Go to any article and click the “save” button in the shareNav menu. The save article modal should appear next to the ShaveNav menu.
6) click the save button again to unsave the article. the unsave article notification should show with the text `"Removed from saved articles in myFT"` and should disappear after 5 seconds.
7) The unsaved notification should only show when the `manageArticleLists` flag is switched on. To ensure that this works, switch the `manageArticleLists` flag off and repeat steps 4 - 6
